install_config/install/advanced_install.adoc

```diff
@@ -1565,6 +1565,10 @@ xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#i
 openshift_metrics_cassandra_storage_type=dynamic
 ----
 
+You can set this variable with CNS 3.6 to enable Gluster
+Block Storage as a backend for logging with ElasticSearch. For more information,
+see link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/container-native_storage_for_openshift_container_platform/apes02[Enabling Gluster Block Storage as Backend for Logging].
+
```

Comment:
This is somewhat misleading. This variable does nothing specific to enabling use of gluster-block, it merely tells the installer to provision a PVC from the cluster default StorageClass. The docs I'm writing go into more accurate detail on this.